### PR TITLE
[WIP] contactbot freshening

### PR DESCRIPTION
**Pull request by itdaniher**
We haven't deployed ContactBot in almost 2 months... a few things have changed since then!

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201632126650504/1201632307208450) by [Unito](https://www.unito.io)

### CODE REVIEW

File: contact/Dockerfile [hunk 1]
```diff
@@ -23,18 +23,14 @@ RUN VIRTUAL_ENV=/app/venv pipenv install
 FROM ubuntu:hirsute
 WORKDIR /app
 RUN mkdir -p /app/data
-RUN apt update
-RUN apt install -y python3.9 wget libfuse2 kmod #npm
+RUN apt update && apt install -y python3.9 wget libfuse2 kmod
 RUN apt-get clean autoclean && apt-get autoremove --yes && rm -rf /var/lib/{apt,dpkg,cache,log}/
 
-# v5.12.2 for fly.io
-RUN wget -q -O fuse.ko "https://public.getpost.workers.dev/?key=01F54FQVAX85R1Y98ACCXT2AGT&raw"
```
Comment (technillogue) [inline contact/Dockerfile:31]: wait, no fuse? I thought we needed that

File: contact/contactbot.py [hunk 2]
```diff
@@ -296,16 +283,17 @@ async def do_make_rule(self, msg: Message) -> Response:
...
             f"values ('{_id}', '{destination}', 'assigned') on conflict (id) do update "
             f"set destination='{destination}', status='assigned' "
         )
+        return f"Set { _id } -> {destination}"
 
     if not utils.get_secret("ORDER"):
         del do_order, do_pay
 
-    async def start_process(self) -> None:
+    async def start_process2(self) -> None:
```
Comment (technillogue) [inline contact/contactbot.py:296]: this will never get run, why is it necessary?

File: contact/Dockerfile (hunk 1 above)
Comment (itdaniher) [reply contact/Dockerfile:31]: no, it's built with dkms afaik

File: contact/contactbot.py (hunk 2 above)
Comment (itdaniher) [reply contact/contactbot.py:296]: "ran fine without it"

File: contact/Dockerfile (hunk 1 above)
Comment (technillogue) [reply contact/Dockerfile:31]: what is kmod about

**Pull request closed without merging**